src/viam/ur/module/utils.cpp

```diff
@@ -6,12 +6,8 @@
 #include <viam/sdk/resource/resource.hpp>
 
 void configure_logger(const viam::sdk::ResourceConfig& cfg) {
-    std::string level_str{};
-    try {
-        level_str = find_config_attribute<std::string>(cfg, "log_level");
-    } catch (...) {
-        level_str = "warn";
-    }
+    auto level_str = find_config_attribute<std::string>(cfg, "log_level").value_or("warn");
```

Comment:
Nice. For some reason I thought `value_or` for `std::optional` wasn't in C++17, but it is!